packages/client/src/views/SelectVitalEvent/SelectVitalEvent.tsx

```diff
@@ -13,11 +13,14 @@ import * as React from 'react'
 import styled, { keyframes } from '@client/styledComponents'
 import { WrappedComponentProps as IntlShapeProps, injectIntl } from 'react-intl'
 import { connect } from 'react-redux'
-import { PrimaryButton } from '@opencrvs/components/lib/buttons'
 import { ErrorText } from '@opencrvs/components/lib/ErrorText'
 import { FixedEventTopBar } from '@opencrvs/components/lib/EventTopBar'
 import { RadioButton } from '@opencrvs/components/lib/Radio'
-import { BodyContent, Container } from '@opencrvs/components/lib/Content'
+import { Frame } from '@opencrvs/components/lib/Frame'
+import { AppBar } from '@opencrvs/components/lib/AppBar'
```

Comment:
This is not the correct use of AppBar, we will address it in this ticket: https://github.com/opencrvs/opencrvs-core/issues/5510